src/main/target/SPRACINGF7DUAL/sound.c

```diff
@@ -0,0 +1,152 @@
+/*
```

Comment:
I think this should go into `src/main/drivers`.

Reply:
I have a new feature that will do exactly that.  new feature in new PR, this just lays the groundwork for experimenters.